src/SFA.DAS.EmployerIncentives.Data/IncentiveApplication/IncentiveApplicationQueryRepository.cs

```diff
@@ -66,6 +67,14 @@ private static IncentiveApplicationApprenticeshipDto MapToApprenticeshipDto(Ince
 
         private static bool MapBankDetailsRequired(string vrfCaseStatus)
         {
+            if (vrfCaseStatus != null && 
+                  (vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedDataValidation, StringComparison.InvariantCultureIgnoreCase)
+                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVer1, StringComparison.InvariantCultureIgnoreCase)
+                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVerification, StringComparison.InvariantCultureIgnoreCase)))
+            {
+                return true;
+            }
+
```

Comment:
would it read a little clearer if we just returned

return (String.IsNullOrWhiteSpace(vrfCaseStatus) ||                (vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedDataValidation, StringComparison.InvariantCultureIgnoreCase)
                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVer1, StringComparison.InvariantCultureIgnoreCase)
                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVerification, StringComparison.InvariantCultureIgnoreCase)));